docs/_docs/reference/experimental/into-modifier.md

```diff
@@ -58,24 +58,63 @@ concatAll(List('a'), "bc", Array('d', 'e'))
 ```
 would apply two _different_ implicit conversions: the conversion from `String` to `Iterable[Char]` gets applied to the second argument and the conversion from `Array[Char]` to `Iterable[Char]` gets applied to the third argument.
 
+Note that a vararg parameter type with into modifiers needs to be put in parentheses, as is shown in the example above. This is to make the precedence clear: each element of the argument sequence is converted by itself.
+
 ## Retrofitting Scala 2 libraries
```

Comment:
There's also the case where a library wants to cross-build between Scala 2 and 3 and wants to avoid feature warnings for conversions in Scala 3 clients. That library could use @into in its API.

Reply:
Such a library wouldn't have `Conversion`s either, though. It would have `implicit def`s.

Reply:
Except if the library also wants to accommodate conversions defined in client code.